### DISCUSSION TITLE
Q&A: hash for all models

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#3298] [Q&A] [answered]
i forget ...
it give an option to start fooocus and generate from all models the hash?

and than next time i start fooocus its all generated, right?

**Comment by mashb1t** [comment, marked as answer]
see release notes, use `--rebuild-hash-cache` only once to generate all hashes

**Reply by mashb1t** [reply]
But if you don't use this the hashes will be generated anyways, but on demand as lazy loading when the hash of the model/LoRA has not yet been generated.
